modules/evm/src/lib.rs

```diff
@@ -217,8 +218,9 @@ decl_storage! {
 			for (address, account) in &config.accounts {
 				let account_id = T::AddressMapping::to_account(address);
 
-				<Accounts<T>>::insert(address, <AccountInfo<T>>::new(account.nonce, None));
-				<Module<T>>::on_contract_initialization(address, &T::AccountId::default(), account.code.clone(), Some(account.storage.len() as u32));
+				let account_info = <AccountInfo<T>>::new(account.nonce, None).expect("Contract is null, it can't fail; qed");
```

Comment:
we can have two version of new, `new_account` and `new_contract` so we don't need the expect here